elasticsearch-extensions/lib/elasticsearch/extensions/test/cluster.rb

```diff
@@ -299,23 +296,25 @@ def start
           def stop
             begin
               nodes = __get_nodes
-            rescue Exception => e
-              __log "[!] Exception raised when stopping the cluster: #{e.inspect}".ansi(:red)
+            rescue StandardError => e
```

Comment:
I believe this is changing behavior of the cluster test. `rescue Exception` would fit best here. 

If Rubocop gives you a hard time, add `# rubocop:disable ___` where ___ is the name of the Cop (rule) you wish to disable.